crates/bindings-csharp/Runtime/bindings.c

```diff
@@ -100,7 +100,11 @@ static MonoArray* stdb_buffer_consume(Buffer buf);
 //   return out;
 // }
 
-__attribute__((import_module("spacetime"),
+#define STDB_ABI_MAJOR_VERSION "6"
+#define STDB_IMPORT_MODULE_MINOR(minor) "spacetime_" STDB_ABI_MAJOR_VERSION "." #minor
+#define STDB_IMPORT_MODULE STDB_IMPORT_MODULE_MINOR(0)

```

Comment:
Nit: let's just inline the string here, same as Rust does. It's a bit weird to edit two parts of version in two places.

Reply:
sure :+1:

Reply:
I meant inline minor into the same string too, so that it's just `spacetime_6.0` like in Rust.